feedback/templates/feedback_tags/import_tags.html

```diff
@@ -0,0 +1,59 @@
+{% extends "manage/base.html" %}
+{% load i18n %}
+{% block content %}
+{% load colortag %}
```

Comment:
I would suggest having this ```{% load colortag %}``` tag before the tag ```{% block content %}```, so the order is more coherent (first loading packages, then only defining the content of the block).